### ISSUE TITLE
How to revert to DVCS Connector?

### ISSUE THREAD

**Issue by jimgraham** [integrations/jira#218]
Hello

Can you point to documentation on how to revert back to the DVCS Connector? This App does not work on our install and we have lost the ability to link Jira Issues to branches and PRs.

We are willing to try this app in the future when you have resolved the reliability issues (see #215, #216, etc) but we cannot go on with the loss of functionality this change has incurred.

Thanks,

**Comment by welps** [comment]
Assuming this is Jira Cloud.

Under https://community.atlassian.com/t5/Jira-articles/Everything-you-need-to-know-about-migrating-from-the-DVCS/ba-p/945910 , it states that you need to reach out to Atlassian to revert back to DVCS Connector.

**Comment by jnraine** [comment]
Hi @jimgraham. I'm sorry to hear the new integration doesn't link your issues to branches and PRs. As @welps mentioned, you can reach out to Atlassian to revert back to the DVCS Connector.

> Lastly, it’s important to note that your GitHub data from the DVCS Integration has not been lost, we just stopped showing it in the UI when you installed the new GitHub for Jira app. This is so there is not duplicate information being displayed. Please contact support.atlassian.com if things have gone horribly wrong and you would like the DVCS Connector integration back and direct help moving over to the new integration. We are happy to help you through this process!

Could you send examples of strings that worked with DVCS Connector—including where they appeared: commits, branch, etc—but doesn't work with the new integration? We're working to fix outstanding bugs and your report will go a long way to make that happen.

**Comment by reillydavis** [comment]
@jimgraham If you don't mind me asking, did your sync ever complete? Also, did you have any issues in getting Atlassian to revert back to DVCS Connector? They have told me it is not possible anymore... and I can't even get the GitHub sync to complete. Really put us in a bad place...

**Comment by jimgraham** [comment]
Hi

We've attempted one more "turn it off / on" again to attempt the resync. The sync was running over the weekend and is still running as of five days later.

We have reached out to Atlassian but haven't heard back.

As for strings that used to work .. my understanding is that it's the sync that's either broken, or not completing, or we're not waiting long enough. Our usual workflow is to put the issue string in the branch name. We always use `ISSUE-text-description` in the branch name and ISSUE is in All Caps at the start of the branch name.

Some examples:

Historical:
```
ADP-58-lock-on-create-account
ADP-53-make-coa-synchronous
UP-6-fix-support-login-cookies
```

Just created this morning
```
IAM-1023-test-for-github-sync
```

It's unclear to me if new branches & PRs will be linked while the historical sync is ongoing.
